apps/coscrad-frontend/src/components/resources/photographs/photograph-detail.spec.tsx

```diff
@@ -30,6 +30,8 @@ const photographToFind: IPhotographViewModel = {
     },
     photographer: 'Johnny Blue',
     imageUrl: 'https://jazzysnaps.images.com/doghouse.png',
+    contributions: [],
+    actions: [],
```

Comment:
I'm just wondering if we are returning the index-scoped actions from the `/photographs` endpoint. I don't think there are any detail-scoped actions.

Reply:
Let's add a story for adding the `CREATE_PHOTOGRAPH` cypress test soon for the dynamic form.